modules/cluster-logging-release-notes-5.4.z.adoc

```diff
@@ -0,0 +1,39 @@
+//Z-stream Release Notes by Version
+[id="cluster-logging-release-notes-5-4-1"]
+= Logging 5.4.1
+This release includes https://access.redhat.com/errata/RHSA-2022:2216[RHSA-2022:2216-OpenShift Logging Bug Fix Release 5.4.1].
+
+[id="openshift-logging-5-4-1-bug-fixes"]
+== Bug fixes
+* Before this update, the log file metric exporter was only reporting logs created while the exporter was running, resulting in inaccurate log growth data. This update resolves this by monitoring /var/log/pods. (https://issues.redhat.com/browse/LOG-2442[LOG-2442])
+
+* Before this update, the collector would be blocked by continually trying to use a stale connection when forwarding logs to fluentd forward receivers.  With this change, the `keepalive_timeout` value has been set to 30s so the collector will recycle the connection and retry failed messages in a more reasonable amount of time. (https://issues.redhat.com/browse/LOG-2534[LOG-2534])
+
+* Before this update, an error in the gateway component enforcing tenancy for reading logs limited access to logs with a Kubernetes namespace causing "audit" and some "infrastructure" logs to be unreadable. With this update, the proxy correctly detects users with admin access and allows access to logs without a namespace. (https://issues.redhat.com/browse/LOG-2448[LOG-2448])
+
+* Before this update, `system:serviceaccount:openshift-monitoring:prometheus-k8s` had cluster level privileges as a `clusterrole` and `clusterrolebinding`. This update restricts the `serviceaccount` to the `openshift-logging` namespace with a role and rolebinding. (https://issues.redhat.com/browse/LOG-2437[LOG-2437])
```

Comment:
```suggestion
* Before this update, `system:serviceaccount:openshift-monitoring:prometheus-k8s` had cluster level privileges set as `clusterrole` and `clusterrolebinding`. This update restricts the `serviceaccount` to the `openshift-logging` namespace with a role and rolebinding. (https://issues.redhat.com/browse/LOG-2437[LOG-2437])
```

Reply:
'as a' is correct here. To say that privileges are 'set as' implies an order of causation that is incorrect.